### DISCUSSION TITLE
The Bourne-again shell for installation

### DISCUSSION THREAD

**Discussion by bobettebob** [Homebrew/discussions#2833] [Getting started] [answered]
I have a MacMini with Monterey 12.1
Xcode 13.2.1 installed by appel.
CLT for Xcode13.2 installed by terminal command
macOS Requirements for Homebrew
How to install The Bourne-again shell?
How to know if it's installed correctly?
Thank you so much for your respond and I hope helping other brand new. :)

**Comment by carlocab** [comment, marked as answer]
The right shell comes with your Mac; there's no need to worry about installing it.
